### ISSUE TITLE
PDF .column-margin error

### ISSUE THREAD

**Issue by PlethoraChutney** [quarto-dev/quarto-cli#5554]
### Bug description

Hello! I am loving quarto so far and am starting to write up my dissertation using it! I'd like to add margin notes, especially because they look so wonderful in the HTML output. However, when I add a div with `.column-margin` to the document I get the error below. I apologize if I've missed anything, or if this isn't a supported use!

Reproducible example:
```
---
format: pdf
---

This paragraph works fine, but the marginpar below throws an error. Both work as expected in HTML output.

::: {.column-margin}
Hello world
:::
```

Quarto reports an error from the LaTeX log file. I've included a few lines above the (bottom) four that Quarto gives since they look relevant:
```
Package marginnote Info: right page because not two side mode on input line 168
.
Runaway argument?
{\if@mn@verbose \PackageInfo {marginnote}{xpos seems to be \@mn@currxpos \ETC.
! Paragraph ended before \rlap was complete.
<to be read again> 
                   \par 
l.168 
```

Information:

| Parameter | Value |
| ----------- | ------ |
| Quarto Version | 1.3.340 |
| IDE | VS Code 1.78.2 (but error persists when quarto is run from command line) |
| OS | Windows 10 Enterprise Version 22H2 build 19045.2965 |

Quarto Check:

```
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.3.340
      Path: C:\Users\Rich\AppData\Local\Programs\Quarto\bin
      CodePage: 1252

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.10.2
      Path: C:/Users/Rich/AppData/Local/Programs/Python/Python310/python.exe
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with py -m pip install jupyter

[>] Checking R installation...........OK
      Version: 4.2.3
      Path: C:/PROGRA~1/R/R-42~1.3
      LibPaths:
        - C:/Users/Rich/AppData/Local/R/win-library/4.2
        - C:/Program Files/R/R-4.2.3/library
      knitr: 1.42
      rmarkdown: 2.20

[>] Checking Knitr engine render......OK
```

### Checklist

- [ ] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.
- [X] Please provide the output of `quarto check` so we know which version of quarto and its dependencies you're running.

**Comment by cscheid** [comment]
>  Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.

Please do include an example.

**Comment by PlethoraChutney** [comment]
I apologize, it was the first code snippet there. Am I meant to attach it somehow?

```
---
format: pdf
---

This paragraph works fine, but the marginpar below throws an error. Both work as expected in HTML output.

::: {.column-margin}
Hello world
:::
```

**Comment by dragonstyle** [comment]
Thanks for reporting this issue - I'm sorry you've run into it!

Not surprisingly, we can't reproduce the issue meaning it is likely something specific to your environment. A couple of ideas:

1) Runaway argument? can sometimes be caused by corrupt intermediate files. Can you try clearing any intermediate files (keep only the qmd and necessary source code) and see whether recompiling without those intermediates solves the problem?

2) Does this happen if you try a document in another directory?

3) Perhaps you could try making sure you have a reasonable version of the 'margin note' pagckage (I ran `tlmgr update marginnote`) 

4) Are there other differences that you could think (templates or anything else in play that might cause the issue)?

**Comment by PlethoraChutney** [comment]
Not surprising to me either! I ran my example on my Mac when I got home and it went fine, so I have to assume it's something about this environment as well.

1. Removed all the files and problem persisted
2. Ran in a different dir and different drive, problem persisted.
3. Ah hah! Running `tlmgr update marginnote` returns 
```
> tlmgr update marginnote

C:\texlive\2019\texmf-dist\scripts\texlive\tlmgr.pl: unexpected return value from verify_checksum: -5
```
I didn't realize I was using my own installation of LaTeX. Running `quarto install tinytex` resolved the issue. Thank you very much!
